pandas/core/reshape/concat.py

```diff
@@ -44,11 +44,16 @@ def concat(objs, axis=0, join='outer', join_axes=None, ignore_index=False,
         Specific indexes to use for the other n - 1 axes instead of performing
         inner/outer set logic
     ignore_index : boolean, default False
-        If True, do not use the index values along the concatenation axis. The
-        resulting axis will be labeled 0, ..., n - 1. This is useful if you are
-        concatenating objects where the concatenation axis does not have
-        meaningful indexing information. Note the index values on the other
-        axes are still respected in the join.
+        If True, do not use the index values along the concatenation axis.
```

Comment:
same.

Reply:
do you mean the ` pd.concat.__doc__` issue?

Reply:
as i said